### DISCUSSION TITLE
Suggestion for RT problem

### DISCUSSION THREAD

**Discussion by kriptofurkangok** [IBAX-io/go-ibax#1808] [Ideas] [answered]
I think there is a back-up plan you should consider;

There are two type of big reward in airdrop rules

1. In 60 days, 60 lucky winners will gets 50.000 IBXC. Total amount is **3 million IBXC**. BUT today no one knows who is the lucky or not. Cancel this distribution and add these coins to general pool.

2. There are **700.000 IBXC** for 7 lucky winners who gets over 45.000 IBAX points. BUT no one reached this score so also add these to general pool.

With this formula, everyone can get ~670-800 IBXC instead of ~50-60. After all this math there is still 500.000 IBXC for top 1000. You can also use that to make fair distrubion. Then everyone is happy from top to bottom. Please think about that for everyone who participate and project's reputation.

- By the way i'm already in top 1000. So if even i care, you should too...

**Comment by m3liksah** [comment]
That's a good idea !

**Comment by yolandadadada** [comment, marked as answer]
Hi, right now we haven't published the point ranking, it's still being cleared, so no one will be able to know their ranking, and the lucky winner's coins will be credited directly to their bound wallet after redemption.
